src/rust/engine/engine_pyo3/src/externs/interface/fs.rs

```diff
@@ -191,7 +189,7 @@ impl PySnapshot {
   }
 
   #[getter]
-  fn files(&self, py: Python) -> Py<PyTuple> {
+  fn files<'py>(&self, py: Python<'py>) -> &'py PyTuple {
```

Comment:
The `Py<>` wrapper meant that the type could be passed without the GIL, but that's not relevant to us. I only didn't do the original because I was still learning lifetimes. See https://pyo3.rs/v0.15.0/types.html#pyt-and-pyobject.